src/GitHub.App/Models/Account.cs

```diff
@@ -30,9 +35,12 @@ public class Account : ReactiveObject, IAccount
             PrivateReposInPlan = privateRepositoryInPlanCount;
             IsOnFreePlan = privateRepositoryInPlanCount == 0;
             HasMaximumPrivateRepositories = OwnedPrivateRepos >= PrivateReposInPlan;
+            CreatedAt = createdAt ?? DateTime.Now;
+            BitmapSource = bitmapSource;
 
-            bitmapSource.ObserveOn(RxApp.MainThreadScheduler)
-                .Subscribe(x => avatar = x);
+            _bitmapSourceSubscription = bitmapSource
+                .ObserveOn(RxApp.MainThreadScheduler)
+                .Subscribe(x => Avatar = x);
```

Comment:
Keep track of the `BitmapSource` `Subscribe()` disposable so we can stop the subscription if we are copied into.